src/version.txt

```diff
@@ -1 +1 @@
-0.2.0
+0.2.0+build.1
```

Comment:
I view the build portion of the version as being used when the code/configuration hasn't changed, but you are simply creating a new release using the same code as before, perhaps to pick up an updated dependency, say.  I'd view these changes as warranting at least a patch bump due to the new version of Python.

I don't want to belabor this point, I just want to make sure that I correctly understand when it is correct to bump the build portion of the version versus patch, minor, or major.

Reply:
Ah, right.  I always forget about that wrinkle.  Thank you for explaining!

Reply:
I created https://github.com/cisagov/skeleton-docker/issues/231 to get that information into the README.